build.gradle

```diff
@@ -66,3 +77,11 @@ dependencies {
     testImplementation group: 'org.mockito', name: 'mockito-core', version:'1.10.19'
     testImplementation group: 'org.hamcrest', name: 'hamcrest-library', version:'1.3'
 }
+
+// Creates a version.txt file containing the current version of the SDK.
+// This file is picked up and parsed by our Ship Orb to determine the version.
+task exportVersion()  {
+    doLast {
+        new File(rootDir, "version.txt").text = "$version"
+    }
+}
```

Comment:
We use this file to be able to know what version to use. It's a lot easier than parsing the build.gradle file in a bash script.